bug-bounty-preview-scopes.md

```diff
@@ -0,0 +1,33 @@
+# .NET Bounty Scope
+
+## Introduction
+
+The [.NET bug bounty](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) covers current released versions of .NET Core, ASP.NET Core and .NET, as well as the latest beta or release candidate of the upcoming version.
+
+.NET has the concept of [preview features](https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md) which ship in the current version, or in nightly builds of the upcoming version. Preview features are not enabled by default, but their goal is to be enabled in the next major release. Preview features are now considered in scope for the bug bounty if they are listed in the table below.
+
```

Comment:
Let's assume for the sake of argument that HTTP/3 support ships proper in .NET 7. How would the "current" release play out in the table below? Do we say that as soon as the feature goes stable and vNext ships, it's now unsupported (thus ineligible for bounty) in .NET 6?

Reply:
Nit: Technically, HTTP/3 is always unsupported in .NET 6, even before .NET 7 ships (it's marked as preview).

I think what you mean to ask is: do preview features in the bug bounty program have the same servicing lifespan as the rest of release? I'd say yes, otherwise it would be hard to follow.

Reply:
That's an interesting question, I don't know. It would depend on what we discovered, but I'd presume that we backport fixes to any preview in an LTS release. This is more of an internal process question though.

Reply:
The challenge is that some of the preview support might require tooling changes (e.g. in the SDK or compiler). For .NET 7 we might have reacted to customer feedback from .NET 6 that requires making changes in the tooling. Since we don't want to support both, the old way and the new way in the .NET 7 SDK, we might not be able to enable customers to use the preview feature on .NET 6 anymore. However, for features like HTTP3 (which doesn't require tooling features) this might not apply. So I guess it's case by case.